Examples/Algorithms/TrackFinding/src/TrackFindingOptions.cpp

```diff
@@ -0,0 +1,35 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "ACTFW/TrackFinding/TrackFindingOptions.hpp"
+
+#include <string>
+
+void FW::Options::addTrackFindingOptions(FW::Options::Description& desc) {
+  using boost::program_options::value;
+
+  auto opt = desc.add_options();
+  opt("ckf-chi2max-slselection", value<double>()->default_value(15),
```

Comment:
I think it might be clearer if the ordering is reversed, i.e. `ckf-slselection-{chi2max,nslsmax}`.

Nitpicking: would `ckf-slsection-nmax` for the section option be enough? `nslsmax` is only understandable if you already know what all the abbreviations are for.

Reply:
That makes more sense. Thank you!